packages/grant-explorer/src/index.tsx

```diff
@@ -54,6 +54,9 @@ const dataLayer = new DataLayer({
   subgraph: {
     endpointsByChainId: getConfig().dataLayer.subgraphEndpoints,
   },
+  indexer: {
+    baseUrl: getConfig().dataLayer.gsIndexerEndpoint as string,
```

Comment:
we shouldn't need to do this cast, something is wrong either in the config definition or elsewhere

if we expect `gsIndexerEndpoint` to not be null or undefined we should check and throw an error instead otherwise it will silently fail downstream potentially causing a hard to debug problem!

Reply:
I see this was just fixed before posting review, awesome